src/analysis/normed/order/basic.lean

```diff
@@ -68,13 +75,23 @@ instance normed_linear_ordered_group.to_normed_ordered_group [normed_linear_orde
 { dist_eq := normed_linear_ordered_field.dist_eq,
   norm_mul' := normed_linear_ordered_field.norm_mul' }
 
+@[priority 100] instance normed_lattice_field.to_normed_lattice_add_comm_group  (α : Type*)
+  [normed_lattice_field α] : normed_lattice_add_comm_group α :=
+⟨λ _ _ h c, add_le_add_left h c, normed_lattice_field.solid⟩
+
 instance : normed_linear_ordered_field ℚ :=
 ⟨dist_eq_norm, norm_mul⟩
 
+instance rat.normed_lattice_field : normed_lattice_field ℚ :=
```

Comment:
```suggestion
instance : normed_lattice_field ℚ :=
```

Reply:
Noted that this part comes from another PR #18554